libethashseal/Ethash.cpp

```diff
@@ -84,10 +84,13 @@ StringHashMap Ethash::jsInfo(BlockHeader const& _bi) const
 
 EVMSchedule const& Ethash::evmSchedule(EnvInfo const& _envInfo) const
 {
-	if (_envInfo.number() >= chainParams().u256Param("frontierCompatibilityModeLimit"))
-		return HomesteadSchedule;
+	if (_envInfo.number() >= chainParams().u256Param("EIP150forkBlock"))
+		return DevconSchedule;
```

Comment:
why git does not update this comment? 
code has been updated to use EIP150Schedule